README.md

```diff
@@ -134,6 +134,13 @@ The code styles which the project uses have been exported as an IntelliJ code st
 
 An Android Studio plugin that provides some quality of life improvements like live templates can be found [HERE](https://github.com/simpledotorg/simple-android-idea-plugin).
 
+## Building an APK with a different build variant
+
+There are currently 2 ways to build an app pointing to different environments:
+
+1. Changing the `qa` API URL in `gradle.properties` file to point to the environment you want. These builds will be debuggable and require us to clone the project and build it using [Android Studio](https://developer.android.com/studio).
```

Comment:
I should probably also add a warning that these changes should not be committed back to the `master` branch?

Reply:
Yeah, I think that's a good thing.

Reply:
Done ✅